tests/cpp/dlr_tensorflow2/dlr_tensorflow2_test.cc

```diff
@@ -198,6 +206,13 @@ void CheckAllDLRMethods(DLRModelHandle& handle, const int batch_size, const int
     // Imagenet 281 - tabby, tabby cat
     EXPECT_GE(output[out_offset + 281], 0.3f);
   }
+
+  // GetDLROutputByName
+  std::vector<float> output_1(out_size);
+  if (GetDLROutputByName(&handle, "logits", output_1.data())) {
+    FAIL() << "GetDLROutputByName failed";
+  }
+  EXPECT_TRUE(std::equal(std::begin(output_1), std::end(output_1), std::begin(output)));
```

Comment:
What if we use `v.begin()` instead of `std::begin(v)`?
```
EXPECT_TRUE(std::equal(output_1.begin(), output_1.end(), output.begin()));
```

Reply:
Updated.